CMake/cdat_modules_extra/setup_runtime.sh.in

```diff
@@ -1,6 +1,18 @@
 # Everything beyond this point will be determined relatively
 # from this path.
 install_prefix="@CMAKE_INSTALL_PREFIX@"
+# Reset previous uvcdat env messing up
+source @CMAKE_INSTALL_PREFIX@/bin/reset_runtime.sh
+
+# Saves what we will mess with
+for v in PATH LD_LIBRARY_PATH DYLD_LIBRARY_PATH PYTHONPATH OPAL_PREFIX LIBOVERLAY_SCROLLBAR ; do
+   tmp="${v}"
+    if [ -n "${!tmp}" ] ; then
```

Comment:
Ahh. That is specific to bash. ZSH uses a different notation for indirect references.

See: https://misterpinchy.wordpress.com/2012/10/16/indirect-variable-references-in-bash-and-zsh/

Perhaps, add a [shebang](http://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang/10383546#10383546) at the beginning of the set/reset scripts.